packages/plugin-global-user-properties/README.md

```diff
@@ -0,0 +1,64 @@
+<p align="center">
+  <a href="https://amplitude.com" target="_blank" align="center">
+    <img src="https://static.amplitude.com/lightning/46c85bfd91905de8047f1ee65c7c93d6fa9ee6ea/static/media/amplitude-logo-with-text.4fb9e463.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# @amplitude/plugin-global-user-properties
+
+Official SDK plugin for adding global user properties to events
+
+## Installation
+
+This package is published on NPM registry and is available to be installed using npm and yarn.
+
+```sh
+# npm
+npm install @amplitude/plugin-global-user-properties
+
+# yarn
+yarn add @amplitude/plugin-global-user-properties
+```
+
+## Usage
+
+This plugin works on top of Amplitude Browser SDK and adds web attribution tracking features to built-in features. To use this plugin, you need to install `@amplitude/plugin-global-user-properties `v0.0.0` or later.
```

Comment:
This was added in other plugins to say that the Browser SDK should be from a certain version. Is that what we meant by this?

Reply:
Looking at the plugin, it doesn't seem that it requires some version of the Browser SDK.

Reply:
adds web attribution tracking features to built-in features?

Reply:
I think I will just delete this - this was copy-pasted from the previous plugin since I wasn't sure which version started plugin support.